docs/setting-up-local-development.md

```diff
@@ -1 +1,286 @@
-*(This page needs to be written.  See [exercism/exercism.io#3251](https://github.com/exercism/exercism.io/issues/3251) for details.)*
+# Setting up Local Development
+
+This section walks you through getting the exercism.io app running locally.
+
+## Prerequisites
+
+### To run the application locally
+
+Backend:
+
+- Ruby
+- PostgreSQL
+
+Frontend:
+
+- Node.js
+
+To install Ruby, check out [RVM](https://rvm.io), [rbenv](https://github.com/sstephenson/rbenv) or [ruby-install](https://github.com/postmodern/ruby-install).
```

Comment:
Perhaps you could link directly to the installation documentation pages? For example, for RVM you could refer to https://rvm.io/rvm/install.